### PR TITLE
You can now remove embedded objects directly with a hemostat

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Getting someone with a bullet or a throwing star lodged in their skull to lay down and let you do the remove embedded object surgery on them (possibly several times if they have shrapnel in multiple limbs) is a lot to ask for in hectic situations, and I doubt people ever really bothered since the object would probably fall out on its own (throwing stars notwithstanding) by the time you could actually finish the surgery.

So, now you just target the affected limb and use a hemostat on the patient to pry shrapnel out, ez pz. It takes a few seconds that scales with the size of the object, and you can do it to yourself if you're fine with taking twice as long (and happen to be carrying around a hemostat, you weirdo). This process causes no harm to the patient.

This PR also makes a few other tiny tweaks to embedding I've been meaning to get around to. Here's the breakdown:

- Objects thrown extra fast (higher than their normal throw_speed, like thrown by a hulk or an amped up disposal outlet) gain +10 to their embedding chance for each extra speed.
- Most bullets are now slightly more likely to embed, especially at point blank
- .357 bullets are somewhat more likely to cause piercing wounds

## Why It's Good For The Game
Makes removing embedded objects from other people less obnoxious. The other changes just make embedding and bullets more consistent

## Changelog
:cl: Ryll/Shaps
add: You can now remove embedded objects in a patient by using a hemostat on the patient's affected limb, no surgery necessary!
del: Removed the now obsolete "remove embedded objects" surgery
balance: Objects thrown at high speed (like by a hulk or amped up disposal outlet) are now more likely to embed, scaling with the speed they're flung into you at
balance: Most bullets are now slightly more likely to embed
balance: .357 bullets are now somewhat more likely to cause piercing wounds
/:cl:

### CODE REVIEW

File: code/__DEFINES/combat.dm [hunk 1]
```diff
@@ -171,6 +171,8 @@ GLOBAL_LIST_INIT(shove_disarming_types, typecacheof(list(
 #define EMBEDDED_JOSTLE_PAIN_MULTIPLIER			1
 ///This percentage of all pain will be dealt as stam damage rather than brute (0-1)
 #define EMBEDDED_PAIN_STAM_PCT					0.0
+///For thrown weapons, every extra speed it's thrown at above its normal throwspeed will add this to the embed chance
+#define EMBED_CHANCE_SPEED_BONUS				10
```
Comment (Jared-Fogle) [inline code/__DEFINES/combat.dm:175]: Remove the tab indent on this, it's not aligned to anything anyway and are hideous.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Minor quip, rest looks good.

**Comment by SkeletalElite** [comment]
make it possible to do with ghetto hemostat alternatives but make it cause damage

**Comment by Tlaltecuhtli** [comment]
> make it possible to do with ghetto hemostat alternatives but make it cause damage

crowbar has hemostat tool flag

**Comment by Ryll-Ryll** [comment]
how the shit are you supposed to clamp blood vessels with a mf'ing crowbar

**Review by Mothblocks — APPROVED** [review]

**Comment by necromanceranne** [comment]
Pulling bullets out with wirecutters sounds awesome.

**Pull request merged**